chainstate/test-suite/src/tests/chainstate_accounting_storage_tests.rs

```diff
@@ -40,6 +42,27 @@ use rstest::rstest;
 use test_utils::random::{make_seedable_rng, Seed};
 use utxo::UtxosStorageRead;
 
+fn get_genesis_pool_data() -> (PoolId, PoolData) {
+    let (
+        genesis_pool_id,
+        genesis_stake_pool_data,
+        _genesis_stake_private_key,
+        genesis_stake_public_key,
+        _genesis_vrf_private_key,
+        genesis_vrf_public_key,
+    ) = get_regtest_genesis_value();
+
+    let genesis_pool_data = PoolData::new(
```

Comment:
`From` is implemented for this type so you can do `let genesis_pool_data = genesis_stake_pool_data.into()`